docs/src/main/asciidoc/security-proactive-authentication.adoc

```diff
@@ -110,6 +110,51 @@ However, some CDI beans may not be ready for use.
 One exception to this solution is a situation when application endpoints are secured with the xref:security-authorize-web-endpoints-reference.adoc#authorization-using-configuration[Authorization using configuration].
 For more information, see the xref:security-authorize-web-endpoints-reference.adoc#policy-active-cdi-request-context[Inject RequestScoped beans into HttpSecurityPolicy] section of the "Authorization of Web endpoints" guide for more information.
 
+=== Make `SecurityIdentity` available in newly activated CDI request context
```

Comment:
```suggestion
=== `SecurityIdentity` injection in Vert.x route handlers
```

Reply:
By the way, it is not really related to the proactive authentication, right ? Just thinking if it should be in some other doc ?

Reply:
well, privately I think this option will proove to be useful in other scenarios as well, but considering I don't have any concrete example, I'll reduce config to the Vert.x route handlers as we agree and we can expand docs as we see users needing it in other scenarios

Reply:
> By the way, it is not really related to the proactive authentication, right ?

Rather _no_ (they are CDI request contex activation issues user experienced with proactive authentication and that is something described by the section of the guide that I enhanced. But you are correct.

> Just thinking if it should be in some other doc ?

I added it here because that's were I'd look for security CDI activation issues, it is one place. but you have correct thought, if you know about better place, let's move it. I don't think it should be in the reactive routes guide because I'd not look there if I write HTTP proxy and before your comments, this feature didn't only spoke about Vert.x route handlers.

Reply:
@michalvavrik 

>well, privately I think this option will proove to be useful in other scenarios as well, but considering I don't have any concrete example, I'll reduce config to the Vert.x route handlers as we agreed and we can expand docs as we see users needing it in other scenarios

Yeah, IMHO, rather than be not very concrete, we can start by talking about the specific situation with handlers and expand, hopefully that will be easier.

As far as the doc is concerned, perhaps, security tips and tricks ?